kube/cron/hof_db_table_replacer.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  {{ if eq .KUBE_NAMESPACE .BRANCH_ENV }}
+  name: hof-db-table-replacer-{{ .DRONE_SOURCE_BRANCH }}
+  {{ else }}
+  name: hof-db-table-replacer
+  {{ end }}
+spec:
+  schedule: "0 0 * * *"
+  jobTemplate:
+    spec:
+      backoffLimit: 10
+      template:
+        metadata:
+          labels:
+            {{ if eq .KUBE_NAMESPACE .BRANCH_ENV }}
+            name: hof-db-table-replacer-{{ .DRONE_SOURCE_BRANCH }}
+            {{ else }}
+            name: hof-db-table-replacer
+            {{ end }}
+        spec:
+          containers:
+          - name: hof-db-table-replacer
+            image: quay.io/ukhomeofficedigital/hof-db-table-replacer@sha256:241e0d20f87f4dbeb988a026d0d84590edf65a4a3b8a1f46dd67592f919297b9
+            imagePullPolicy: Always
+            securityContext:
+              runAsNonRoot: true
+            envFrom:
+              - configMapRef:
+                  {{ if eq .KUBE_NAMESPACE .BRANCH_ENV }}
+                  name: configmap-{{ .DRONE_SOURCE_BRANCH }}
+                  {{ else }}
+                  name: configmap
+                  {{ end }}
+            env:
```

Comment:
I will send you a list of env vars we will need here. There are a few more than initially in this file.